configure

```diff
@@ -508,6 +523,10 @@ def configure_arm(o):
   o['variables']['arm_thumb'] = 0      # -marm
   o['variables']['arm_float_abi'] = arm_float_abi
 
+  if options.dest_os == 'android':
+    o['variables']['arm_fpu'] = 'vfpv3'
+    o['variables']['arm_version'] = '7'
```

Comment:
Okay, fair enough.  If you can remove the unused function and squash the commits into a single commit with a nice commit log (see `git log configure` for examples), I'll land it for you.